changelog

```diff
@@ -130,6 +130,11 @@ Version 1.13.5+dev:
        The blocking behavior can be reenabled by setting pass_allied_units=no
        in the [tunnel] tag.
  * Lua API:
+   * Upgrade to Lua 5.3.3+
+     Consult http://www.lua.org/ for a full change list and updated documenation.
+     * compatibility with Lua 5.0, 5.1 and 5.2 removed
+     * bit32 functions removed
```

Comment:
Ah okay. If bit32 wasn't in 1.12 then we don't need to leave it in.

Note that bitshift operators cannot be used in typical WML-embedded Lua (or the right-shift one, at least) since they'd be interpreted as end-of-string. I think this probably doesn't matter too much though.